ci/blackbox/rdsbroker_test.go

```diff
@@ -696,6 +696,10 @@ var _ = Describe("RDS Broker Daemon", func() {
 		Describe("Postgres 12", func() {
 			TestRestoreFromPointInTime("postgres", "postgres-micro-12")
 		})
+
+		Describe("MySQL 8.0", func() {
```

Comment:
Why no MySQL 5.7 here?

Reply:
Good question. I think I saw only one pg version being tested and thought I'd do the same for mysql, assuming it was to limit the runtime.

Reply:
Added it for now so we can at least prove it's working, may remove before merge if it's too slow.